Lib/textwrap.py

```diff
@@ -139,6 +145,7 @@ def __init__(self,
         self.max_lines = max_lines
         self.placeholder = placeholder
 
+        self.len = cjklen if self.cjk else len
```

Comment:
Exposing functions mean that you must document them, write unit tests, and that someone (maybe not you) have to maintain them. I don't think that it's worth it. Let's try with something simple, make these functions private.